helm/frost-server/values.yaml

```diff
@@ -95,11 +111,14 @@ frost:
     requiredHostname:
 
     # Common FROST-Server Database properties, shared by any other FROST-Server modules
+    driver: "org.postgresql.Driver"
     implementationClass: "de.fraunhofer.iosb.ilt.sta.persistence.postgres.longid.PostgresPersistenceManagerLong"
     idGenerationMode: "ServerGeneratedOnly"
     database: sensorthings
     username: c2Vuc29ydGhpbmdz
     password: bm93eW91Y2FuY2hhbmdlaXQ=
+    jndi:
+      datasource:
```

Comment:
Better to leave JNDI alone, it's a Tomcat internal thing, and makes not much sense in a Docker environment.